code/modules/mob/living/carbon/human/physiology.dm

```diff
@@ -28,5 +28,7 @@
 
 	var/hunger_mod = 1		//% of hunger rate taken per tick.
 
+	var/work_success_mod = 1 //% Modifier to the success rate of works
```

Comment:
But why is it in physiology at all?

Reply:
It varies depending on person and I saw the other similar mods in it so I placed it there. Where else should I put it?